gradle.properties

```diff
@@ -13,4 +13,4 @@
 #   permissions and limitations under the License.
 #
 
-version=1.6
+version=1.11.0
```

Comment:
What is this for?

Reply:
Actually this variable is not in use. It was pointing to older version. So i updated to latest version. This variable could be used in build.gradle file.

Reply:
I see makes sense.